src/app/shell/personal-cabinet/applications/application-card/application-card.component.ts

```diff
@@ -20,20 +16,24 @@ export class ApplicationCardComponent implements OnInit {
 
   readonly applicationStatusUkr = ApplicationStatusUkr;
```

Comment:
let use some neutral naming so we can easily add supporting of eng without refactoring